publish-scripts/shared/constants.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python3
 
 # same for all different OSes
-PACKAGENAME = "azure-functions-core-tools-4"
+PACKAGENAME = "azure-functions-core-tools"
```

Comment:
I think this constant is shared with the ubuntu installer, and I don't think that was discussed. Do we also want to change the name for ubuntu? That might be fine, but it would mean more testing/doc changes/etc.

For example: https://github.com/Azure/azure-functions-core-tools#2-install

Reply:
Thanks for the catch, @ejizba . I will continue this discussion offline with Anatoli.